docs/features/wait/walk.md

```diff
@@ -0,0 +1,12 @@
+# Walk
+
+Walk walks the strategies tree and calls the visit function for each node.
```

Comment:
question: I see the value in walking though the strategies, could you elaborate more about the use cases in the docs? 🙏